subsys/bluetooth/host/conn.c

```diff
@@ -1864,12 +1873,33 @@ const bt_addr_le_t *bt_conn_get_dst(const struct bt_conn *conn)
 	return &conn->le.dst;
 }
 
-int bt_conn_get_info(const struct bt_conn *conn, struct bt_conn_info *info)
+int bt_conn_get_info(struct bt_conn *conn, struct bt_conn_info *info)
 {
+	if (conn->state != BT_CONN_CONNECTED) {
+		return -ENOTCONN;
+	}
```

Comment:
Minor: add an empty line after `}`

Reply:
Why doen't checkpatch detect this?

Reply:
Because it's not that strict. This is mostly about keeping consistency with the rest of the code in the Bluetooth subsystem.